docs/installing_deis/index.rst

```diff
@@ -1,19 +1,18 @@
 :title: Installing Deis
-:description: Step-by-step guide for operations engineers setting up a private PaaS using Deis.
+:description: Step-by-step guide for operations engineers setting up Deis.
 
 .. _installing_deis:
-.. _provision-controller:
 
 Installing Deis
 ===============
 
-Provisioning Deis is a matter of creating one or more :ref:`concepts_coreos`
-machines and using :ref:`install_deisctl` to install and start Deis.
+To install Deis consists of provisioning three or more :ref:`concepts_coreos`
```

Comment:
Hmm, this reads weird to me. Maybe "Installing Deis consists of provisioning..."?